README.rst

```diff
@@ -19,6 +19,8 @@ Hopefully this is enough to get you going::
     -v, --verbose
     -d, --dump
     -u UUID, --uuid=UUID  Camera UUID
+    -i id, --id=id        Camera Connection id
+    -c, --connect_with_id Connect using id instead of UUID (NVR Beta API)
```

Comment:
I will take a look... There is also changes in the login and getsnapshot for cameras... I have login working but have not figured out the right api call to get snapshot from the camera. The fallback of getting a snapshot through the nvr still works, but the homeassistant module does not have that fallback implemented. 

So I will try to find some similar information from the cameras themselves to determine how we should log in and take snapshots.